packages/ts-compiler/src/typescript-compiler.ts

```diff
@@ -23,18 +23,13 @@ export class TypescriptCompiler implements Compiler {
     const deps = this.preset.getDynamicPackageDependencies ? this.preset.getDynamicPackageDependencies() : {};
     return deps;
   }
+
   public getDynamicConfig(ctx: CompilerContext) {
-    const defaultConfig = {
-      tsconfig: {},
-      development: false,
-      copyPolicy: {
-        ignorePatterns: ['package.json', 'package-lock.json', 'tsconfig.json'],
-        disable: false
-      },
-      useExperimentalCache: false
-    };
-    const presetConfig = this.preset.getDynamicConfig ? this.preset.getDynamicConfig() : {};
-    const config = merge({}, defaultConfig, presetConfig, ctx.rawConfig);
+    //this.preset.getDynamicConfig receive CompilationContext type, a this function receive CompilerContext, we need to update the type of this function to receive GenericObject.
+    //CompilationContext > CompilerContext > rawConfig
+    //@ts-ignore
```

Comment:
fix me

Reply:
I replaced `ts-ignore` by `as any`, but it throws me an error so I put back `ts-ignore`.
The error was because of the function `getDynamicConfig` is not receiving any param of any type.
I'll update immediately the interface of Preset when the code will be merged.